src/bci_build/package.py

```diff
@@ -2564,6 +2564,18 @@ def _get_minimal_kwargs(os_version: OsVersion):
     for os_version in ALL_NONBASE_OS_VERSIONS
 ]
 
+KERNEL_MODULE_CONTAINERS = [
+    OsContainer(
+        name="kernel-module-toolkit",
```

Comment:
Can we name it a bit more specific? I think something like sle15 should be in the name to make it clear that its not for ANY kernel